src/cli/cli.cpp

```diff
@@ -2001,12 +2001,12 @@ void Interpreter::ProcessPollPeriod(int argc, char *argv[])
 
     if (argc == 0)
     {
-        mServer->OutputFormat("%d\r\n", (otLinkGetPollPeriod(mInstance) / 1000)); // ms->s
+        mServer->OutputFormat("%d\r\n", otLinkGetPollPeriod(mInstance));
     }
     else
     {
         SuccessOrExit(error = ParseLong(argv[0], value));
-        otLinkSetPollPeriod(mInstance, static_cast<uint32_t>(value * 1000)); // s->ms
+        otLinkSetPollPeriod(mInstance, static_cast<uint32_t>(value));
```

Comment:
Allowed minimal data poll rate should be 10ms (OPENTHREAD_CONFIG_MINIMUM_POLL_PERIOD).

How about adding check for the value and default as 10ms for not allowed values?
https://github.com/openthread/openthread/blob/master/src/core/thread/data_poll_manager.cpp#L166

Reply:
Ok, updated in the second commit

Reply:
@librasungirl, where were you suggesting to add this check?

Reply:
planed to submit a separate PR to enhance this check.
Just submitted #3401, please have a review.